constants.bzl

```diff
@@ -3,8 +3,8 @@ RELEASE_ENGINES_NODE = "^20.19.0 || ^22.12.0 || >=24.0.0"
 RELEASE_ENGINES_NPM = "^6.11.0 || ^7.5.6 || >=8.0.0"
 RELEASE_ENGINES_YARN = ">= 1.13.0"
 
-NG_PACKAGR_VERSION = "^20.0.0-next.0"
-ANGULAR_FW_VERSION = "^20.0.0-next.0"
+NG_PACKAGR_VERSION = "^20.0.0"
+ANGULAR_FW_VERSION = "^20.0.0"
 ANGULAR_FW_PEER_DEP = "^20.0.0 || ^20.0.0-next.0"
```

Comment:
You also need to update the below two lines. `|| ^20.0.0-next.0` should be removed